### PR TITLE
adds how to configure the registry to send cloud events

### PR DESCRIPTION
**Pull request by smccarthy-ie**

### CODE REVIEW

**Comment by famartinrh** [comment]
the last link added to cloud events demo app is not correct, that app is not related to the features documented here

File: docs/modules/ROOT/partials/getting-started/proc-configuring-registry-events.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
++
+Alternatively, you can set the properties for the kafka producer using the `registry.events.kafka.config` prefix, for example:
+`registry.events.kafka.config.bootstrap.servers=my-kafka-host:9092`
+
+. If required, you can also set the Kafka topic partition to use to produce events:
+** `registry.events.kafka.topic-partition=1`
+
+[role="_additional-resources"]
+.Additional resources
+* For an open source example application, see the https://github.com/Apicurio/apicurio-registry-examples/tree/2.0.x/cloudevents[Apicurio Registry CloudEvents demo]
```
Comment (famartinrh) [inline docs/modules/ROOT/partials/getting-started/proc-configuring-registry-events.adoc:58]: this link should be removed, that app is not related to the featureds being documented

**Comment by smccarthy-ie** [comment]
Thanks @famartinrh. Demo link removed. Merging now...
